lib/chef/audit/runner.rb

```diff
@@ -79,11 +79,15 @@ def require_deps
         require 'rspec'
         require 'rspec/its'
         require 'specinfra'
+        require 'specinfra/helper'
+        require 'specinfra/helper/set'
         require 'serverspec/helper'
         require 'serverspec/matcher'
         require 'serverspec/subject'
         require 'chef/audit/audit_event_proxy'
         require 'chef/audit/rspec_formatter'
+
+        Specinfra::Backend::Cmd.send(:include, Specinfra::Helper::Set)
```

Comment:
The only clue I could glean about what's up was this commit, which made me deduce the absence was intentional and led to me putting this here. 
https://github.com/serverspec/specinfra/commit/8b80f424650255e69981ceeaf8f321d7e2222d14

"set" is obviously a common term,